### PR TITLE
Refinements/remove warning errors

### PR DESCRIPTION
**Pull request by realgcoding**

### CODE REVIEW

**Review by ShadabFaiz — CHANGES_REQUESTED** [review]
2 things to note here.
1.  The PR is outdated. Update it from `development` branch.
2. Even though the test cases are being passed. There are some errors during running test case related to event being fire.

**Review by anirudh411 — APPROVED** [review]

File: src/components/Deliverable/DeliverableTarget.tsx [hunk 1]
```diff
@@ -34,24 +36,65 @@ function getInitialValues(props: DeliverableTargetProps) {
...
+			onCancel();
+		} catch (error) {
+			notificationDispatch(setErrorNotification("Deliverable Target creation Failed !"));
+		}
+	};
+
+	//  fetching category_unit id and on completion creating deliverable target
+	const [getCategoryUnit] = useLazyQuery(GET_CATEGORY_UNIT, {
+		onCompleted(data) {
+			if (data.deliverableCategoryUnitList && data.deliverableCategoryUnitList.length) {
```
Comment (anirudh411) [inline src/components/Deliverable/DeliverableTarget.tsx:90]: can be simplified with `?`

File: src/components/Deliverable/DeliverableTarget.tsx [hunk 2]
```diff
@@ -60,25 +103,24 @@ function DeliverableTarget(props: DeliverableTargetProps) {
 			deliverableTargetForm[1].getInputValue = setcurrentCategory;
 		}
 	}, [deliverableCategories]);
+
 	// handling category change
 	useEffect(() => {
+		console.log("hey", currentCategory);
```
Comment (anirudh411) [inline src/components/Deliverable/DeliverableTarget.tsx:109]: remove console logs

**Review by anirudh411 — CHANGES_REQUESTED** [review]

**Review by anirudh411 — APPROVED** [review]

File: src/components/Deliverable/DeliverableTarget.tsx (hunk 1 above)
Comment (ShadabFaiz) [reply src/components/Deliverable/DeliverableTarget.tsx]: done

File: src/components/Deliverable/DeliverableTarget.tsx (hunk 2 above)
Comment (ShadabFaiz) [reply src/components/Deliverable/DeliverableTarget.tsx]: done

**Review by anirudh411 — APPROVED** [review]

**Review by ShadabFaiz — APPROVED** [review]
Needs alot of fixes...but accepting it now..

**Pull request merged**